shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/ApiArrivalsWithSplits.scala

```diff
@@ -22,25 +22,25 @@ case class ApiFlightWithSplits(apiFlight: Arrival, splits: Set[Splits], lastUpda
     with Updatable[ApiFlightWithSplits]
     with WithLastUpdated {
 
-  def totalPaxFromApi: Option[Int] = splits.collectFirst {
+  def totalPaxFromApi: Option[TotalPaxSource] = splits.collectFirst {
     case splits if splits.source == ApiSplitsWithHistoricalEGateAndFTPercentages =>
-      Math.round(splits.totalPax).toInt
+      TotalPaxSource(Math.round(splits.totalPax).toInt, ApiFeedSource, Option(ApiSplitsWithHistoricalEGateAndFTPercentages))
   }
 
-  def totalPaxFromApiExcludingTransfer: Option[Int] =
+  def totalPaxFromApiExcludingTransfer: Option[TotalPaxSource] =
     splits.collectFirst { case splits if splits.source == ApiSplitsWithHistoricalEGateAndFTPercentages =>
-      Math.round(splits.totalExcludingTransferPax).toInt
+      TotalPaxSource(Math.round(splits.totalExcludingTransferPax).toInt, ApiFeedSource, Option(ApiSplitsWithHistoricalEGateAndFTPercentages))
     }
 
-  def pcpPaxEstimate: Int =
-    totalPaxFromApiExcludingTransfer match {
-      case Some(apiTotal) if hasValidApi => apiTotal
+  def pcpPaxEstimate: TotalPaxSource =
+    totalPaxFromApi match {
+      case Some(totalPaxSource) if hasValidApi => totalPaxSource
       case _ => apiFlight.bestPcpPaxEstimate
     }
 
-  def totalPax: Option[Int] =
+  def totalPax: Option[TotalPaxSource] =
     if (hasValidApi) totalPaxFromApi
-    else apiFlight.ActPax
+    else apiFlight.ActPax.map(TotalPaxSource(_, AclFeedSource, None))
```

Comment:
This shouldn't be `AclFeedSource`. We need check which feed sources the arrival has and set this according to the best source on the following order of precedence:
 - LiveFeedSource
 - ForecastFeedSource
 - AclFeedSource